frontend/src/screens/apps/AppList.tsx

```diff
@@ -8,13 +8,15 @@ import AppCard from "src/components/connections/AppCard";
 import { Button } from "src/components/ui/button";
 import { useApps } from "src/hooks/useApps";
 import { useInfo } from "src/hooks/useInfo";
+import { useIsDesktop } from "src/hooks/useMediaQuery";
 import { useUnusedApps } from "src/hooks/useUnusedApps";
 
 const albyConnectionName = "getalby.com";
 
 function AppList() {
   const { data: apps } = useApps();
   const { data: info } = useInfo();
+  const isDesktop = useIsDesktop();
```

Comment:
Minor: we could use tailwind breakpoints rather than this hook. Not sure which is better or if it really matters CC @reneaaron

Reply:
I'd vote for the tailwind classes as all other responsive stuff (columns, grids, etc) is also implemented that way.

Reply:
It was used first in Credenza but then we removed that (and we are fine without it now ig?), so we can replace that with tailwind classes directly, will do that in this PR itself

Reply:
Done!